### PR TITLE
Add time display to scoreboard

### PR DESCRIPTION
**Pull request by TomyLobo**
Plus some cleanup nearby

screenshot
^ That's what the font looks like in comparison
With mp_timelimit <= 0, it says, for example, "Time: 123 minute(s)"
With mp_timelimit > 0, it says, for example, "Time: 12/34 minute(s)"

### CODE REVIEW

File: mp/src/game/client/sdk/hud/sdk_scoreboard.cpp [hunk 1]
```diff
@@ -427,120 +427,78 @@ void CSDKScoreboard::UpdatePlayerInfo()
...
-	pMostLabelPlayer = dynamic_cast<Label *>(FindChildByName("MostStreakPlayer"));
-	if (pMostLabelPlayer)
-	{
-		C_SDKPlayer* pPlayer = ToSDKPlayer(UTIL_PlayerByIndex(sdkPR->GetHighestKillStreakPlayer()));
-		if (pPlayer)
-			pMostLabelPlayer->SetText(pPlayer->GetPlayerName());
-		else
-			pMostLabelPlayer->SetText("");
-	}
+	UpdateMostLabel(
```
Comment (BSVino) [inline mp/src/game/client/sdk/hud/sdk_scoreboard.cpp]: Nice refactor

File: mp/src/game/client/sdk/hud/sdk_scoreboard.cpp [hunk 2]
```diff
@@ -466,6 +466,18 @@ void CSDKScoreboard::UpdatePlayerInfo()
...
 	);
+
+	Label *pTimeLimitLabel = dynamic_cast<Label *>(FindChildByName("TimeLimit"));
+	if (pTimeLimitLabel)
+	{
+		int iTimeElapsed = SDKGameRules()->GetMapElapsedTime() / 60;
+		int iTimeLimit = mp_timelimit.GetInt();
+
+		wchar_t wszNewLabel[100];
+		V_swprintf_safe(wszNewLabel, iTimeLimit > 0 ? L"Time: %i/%i minute(s)" : L"Time: %i minute(s)", iTimeElapsed, iTimeLimit);
```
Comment (BSVino) [inline mp/src/game/client/sdk/hud/sdk_scoreboard.cpp]: Load this string from i18n string.

Recommend rephrasing for clarity: "Play time: n minutes" vs "Time remaining: n minutes"

**Comment by TomyLobo** [comment]
> Load this string from i18n string.

> Recommend rephrasing for clarity: "Play time: n minutes" vs "Time remaining: n minutes"

Fixed both issues now.

**Comment by BSVino** [comment]
Looks good, ship. :)

**Pull request merged**